server/data/quotes.json

```
@@ -47,6 +47,10 @@
     "author": "Armenian Proverb",
     "sentence": "You are as many a person as languages you know."
   },
+  {
+    "author": "Avril Lavigne",
+    "sentence": "As humans we speak one language..."
```

Please end the sentence with a period.
```suggestion
    "sentence": "As humans we speak one language."
```

---

thanks @Kvaibhav01